temporalcli/commandsmd/commands.md

```diff
@@ -633,6 +633,7 @@ The `temporal schedule list` command lists all Schedules in a namespace.
 
 * `--long`, `-l` (bool) - Include detailed information.
 * `--really-long` (bool) - Include even more detailed information that's not really usable in table form.
+* `--query`, `-q` (string) - Filter results using a SQL-like query.
```

Comment:
Looks like we sometimes document query as SQL-like query and other times as "list filter".
I would go with the latter since that's the terminology we use in the docs: https://docs.temporal.io/visibility#list-filter.

CC @josh-berry in case you want to make this consistent in other places.

Reply:
Changed; thanks